client/assets/styles/sass/_project.scss

```diff
@@ -23,6 +23,10 @@
     font-size: 0.90rem;
   }
 
+  .emailWarning {
```

Comment:
Can't we use the same formatting as we use for other error messages with `.error`?

Reply:
@xamanu I have used this to highlight the link. What do u think?  Thanks